LayoutTests/fast/webgpu/nocrash/fuzz-298315-expected.txt

```diff
@@ -0,0 +1,4 @@
+CONSOLE MESSAGE: 0,0
+CONSOLE MESSAGE: 414141,0,0,0,66000000,0,0,0
```

Comment:
for some reason EWS is showing:
```
CONSOLE MESSAGE: 414141,0,0,0,66000002,0,0,0
```
or
```
CONSOLE MESSAGE: 414141,0,0,0,66000001,0,0,0
```
instead

Reply:
hmm this would mean that the provoking vertex is not always the first vertex_id

Reply:
oh that's due to `primitive: {topology: 'point-list'},` - test is using points and not triangles

That makes sense then, in which case the test expectations differ due to a race condition